lib/stove-testing-e2e-elasticsearch/src/test/kotlin/com/trendyol/stove/testing/e2e/elasticsearch/ElasticsearchTestSystemTests.kt

```diff
@@ -0,0 +1,49 @@
+package com.trendyol.stove.testing.e2e.elasticsearch
+
+import com.trendyol.stove.testing.e2e.database.DocumentDatabaseSystem.Companion.shouldGet
+import com.trendyol.stove.testing.e2e.system.TestSystem
+import com.trendyol.stove.testing.e2e.system.abstractions.ApplicationUnderTest
+import io.kotest.core.config.AbstractProjectConfig
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+
+class Setup : AbstractProjectConfig() {
+    override suspend fun beforeProject() {
+        TestSystem()
+            .withElasticsearch("index-name")
+            .applicationUnderTest(NoOpApplication())
+            .run()
+    }
+
+    override suspend fun afterProject() {
+        TestSystem.instance.close()
+    }
+}
+
+class NoOpApplication : ApplicationUnderTest<Unit> {
+    override suspend fun start(configurations: List<String>) {
+    }
+
+    override suspend fun stop() {
+    }
+}
+
+class CouchbaseTestSystemTests : FunSpec({
+
+    data class ExampleInstance(
+        val id: String,
+        val description: String,
+    )
+    test("should save and get") {
+        val exampleInstance = ExampleInstance("1", "1312")
+        val indexName = "index-name"
+
+        TestSystem.instance
+            .withElasticsearch(indexName)
```

Comment:
`withElasticsearch` belongs to set-up time, not the test time, you may delete it.